sycl/include/sycl/ext/oneapi/experimental/current_device.hpp

```diff
@@ -0,0 +1,38 @@
+//==--------- current_device.hpp - sycl_ext_oneapi_current_device ----------==//
```

Comment:
Even though I like the proposed state better, the extension spec does **not** document a separate header, meaning that this functionality should be available from `sycl/sycl.hpp` directly.